uint8 port
int32 id
int32 status
